spec/support/omniauth_macros.rb

```diff
@@ -6,6 +6,10 @@ module OmniauthMacros
   Rails.application.secrets.google_secret = 'test secret google'
   Rails.application.secrets.facebook_key = 'test key facebook'
   Rails.application.secrets.facebook_secret = 'test secret facebook'
+  Rails.application.secrets.suse_key = 'test key suse'
+  Rails.application.secrets.suse_secret = 'test secret suse'
+  Rails.application.secrets.github_key = 'test key github'
+  Rails.application.secrets.github_secret = 'test secret github'
```

Comment:
There is no '.env.test' yet, if we want to add it then we can move these in there.

Are we aware of any reason why a '.env.test' file hasn't been used so far?